firerunner/bins/controller/controller.rs

```diff
@@ -194,15 +202,15 @@ impl Inner {
                     // Evict an idle VM running some other functions
                     None => {
 //                        println!("No free resources, picking a VM to evict");
-                        if let Some((evict_vm, evict_cpu, evict_mem)) = self.get_evictable_vm(&req) {
+                        if let Some((evict_vm, _, evict_mem)) = self.get_evictable_vm(&req) {
 
```

Comment:
So we will never evict a VM of a different size, meaning a screwed workload with favor functions with less popular memory requirements. I think that's fine for the paper, but let's make an issue to fix after.